authors/authors.json

```diff
@@ -24,6 +32,14 @@
         "sympy_commits": 263
     },
     {
+        "email": "scolobb@colimite.fr",
+        "github_id": "scolobb",
+        "institution": "??",
+        "institution_address": "??",
+        "name": "Sergiu Ivanov",
+        "sympy_commits": 260
+    },
```

Comment:
Thanks @scolobb, I didn't know when you will update it, so I separate your update from the rest into #152.